release/mk-cfgman-docs.sh

```diff
@@ -6,14 +6,25 @@
 
 . ~/.server.config || exit $?
 
-# Known Bug:
-#  Does not remove references to directives dropped. This is
-#  an issue for branch 'master' directive erasure.
+# clean the current workspace
+gitCleanWorkspace ()
+{
+  git clean --quiet -xdf --exclude="\.BASE"
+  git checkout --quiet -- .
+}
 
 for version in `ls -1 $SQUID_VCS_PATH | grep squid | cut -d- -f2`; do
 
-	! test -d $SQUID_WWW_PATH/content/Versions/$version/cfgman && continue
+	# Update the website /Versions/v*/cfgman/ HTML documents
+	cd $SQUID_VCS_PATH/squid-$version || continue
+	gitCleanWorkspace
+	git checkout v$version &&
```

Comment:
If this script relies on something else fetching changes from the remote repository, I recommend adding a comment about it. Otherwise, it needs to do something like "git fetch".